backtesting/_plotting.py

```diff
@@ -220,11 +220,11 @@ def plot(*, results: pd.Series,
 
     pad = (index[-1] - index[0]) / 20
 
-    fig_ohlc = new_bokeh_figure(
```

Comment:
add explanations pls with links to other places, if we need to fix it again later

Reply:
not our changes